nextstrain/cli/util.py

```diff
@@ -628,6 +630,8 @@ def print_runner_tests(tests: RunnerTestResults):
 
         print(status.get(result, str(result)) + ":", formatted_description)
 
+        yield (description, result)
```

Comment:
Move this above the `print()` (to first line of loop body probably) to make fully true the docstring claim:

> Messages will only be printed if the generator is consumed downstream.

(as otherwise the first message will always be printed) and to also match the order of operations implied by the function name.

Reply:
Done:

https://github.com/nextstrain/cli/blob/f6ddf6e139fd56aa63568955901c11786717e0fd/nextstrain/cli/util.py#L631-L633

though order matters less now with https://github.com/nextstrain/cli/pull/418#discussion_r2010958707:

https://github.com/nextstrain/cli/blob/0597de130942e1a66b238b992def830d660e3083/nextstrain/cli/util.py#L631-L633